Uchu.World/Handlers/Commands/CharacterCommandHandler.cs

```diff
@@ -777,9 +779,9 @@ public string World(string[] arguments, Player player)
                 return $"Can't find world with ID {id}";
             }
 
-            string WorldName = WorldTable.ZoneName;
+            var path = Path.Combine(UchuServer.Config.ResourcesConfiguration?.GameResourceFolder, Path.Combine("maps", WorldTable.ZoneName.ToLower()));
```

Comment:
Just wondering, why was this buggy before and how does this fix it? Not that this is wrong or something, I'm just out of the loop.

Reply:
Basically the Zone tables are massive messes there are almost no ways to filter whether a world is correct or not but this does filter out most invalid zones

Reply:
Seems fair.